com.unity.render-pipelines.universal/ShaderLibrary/Lighting.hlsl

```diff
@@ -627,21 +627,146 @@ half3 SampleLightmap(float2 lightmapUV, half3 normalWS)
 #define SAMPLE_GI(lmName, shName, normalWSName) SampleSHPixel(shName, normalWSName)
 #endif
 
-half3 GlossyEnvironmentReflection(half3 reflectVector, half perceptualRoughness, half occlusion)
+half3 BoxProjectedCubemapDirection(half3 reflectionWS, half3 positionWS, float4 cubemapPositionWS, float4 boxMin, float4 boxMax)
+{
+    // Is this probe using box projection?
+    [branch] if (cubemapPositionWS.w > 0.0)
```

Comment:
I used PIX to gpu capture using DX12 with and without the branch tag and there was no real performance difference.
Without the branch tag was the branch removed from the compiled shader.

Because there was no measurable performance benefit, we have chosen to remove the tag so the compiler can optimize the code and take advantage of future compiler optimizations which would otherwise be more restricted.